__tests__/app.test.js

```diff
@@ -251,4 +250,84 @@ describe('API',() => {
         });
     });
   }); // End of 7. POST /api/articles/:article_id/comments
+
+
+  describe("8. PATCH /api/articles/:article_id", () => {
+    test("200, Respond with an update articles objects (in array)", () => {
+      const article = {
+        inc_votes: 100,
+      };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(article)
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toHaveLength(1);
+          expect(article[0]).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: expect.any(String),
+            votes: 200,
+          });
+        });
+    });
+
+    test("404, non-existent article id", () => {
+      const article = {
+        inc_votes: 100,
+      };
+      return request(app)
+        .patch("/api/articles/100")
+        .send(article)
+        .expect(404)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toBe("ID Not Exist");
```

Comment:
Could you change this to " ID does not exist" or "non existent ID" make sure to keep this consistent throughout please :)